src/functions-and-arrays.js

```diff
@@ -1,20 +1,172 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers(num1, num2){
+  return Math.max(num1, num2);
+}
+
 // Iteration #2: Find longest word
+// -- Declare a function named findLongestWord that takes as an argument an array of words and returns the longest one. 
+// -- If there are 2 with the same length, it should return the first occurrence.
+
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-// Iteration #3: Calculate the sum
+function findLongestWord(arrayWords) {
+  if (arrayWords.length === 0) {
+    return null;
+  }
+
```

Comment:
Para limpiar un poco esta funcion podriamos usar un for of loop, que son mas declarativos y retirar la variable largura.

`
 let palabraMasLarga = arrayWords[0]
  for(let word of arrayWords) {
    if(word.length > palabraMasLarga.length) palabraMasLarga = word
}
`